modules/loader.js

```diff
@@ -769,7 +769,7 @@ __private.createSnapshot = height => {
 
 	const snapshotRound = library.config.loading.snapshotRound;
 	const totalRounds = Math.floor(height / ACTIVE_DELEGATES);
-	const targetRound = Number.isNaN(snapshotRound)
+	const targetRound = Number.isNaN(parseInt(snapshotRound))
```

Comment:
Maybe `typeof snapshotRound === 'number'`?

Reply:
Ideal solution would to be to use Bignumber library, so we should refactor in seperate scope.